frontend/.prettierignore

```diff
@@ -0,0 +1,19 @@
+# dependencies
+/node_modules
```

Comment:
This file is a replication of the .gitignore file for the frontend as I was uncertain which files we might want to ignore, and felt this was a good representation.